specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2019-03-01-preview/savedquery.json

```diff
@@ -0,0 +1,537 @@
+      "get": {
+        "description": "Get all saved query defined within a specified subscription and resource group.",
+        "operationId": "SavedQuery_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list containing 0 or more saved queries.",
+            "schema": {
+              "$ref": "#/definitions/SavedQueryListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/SavedQueryError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "Get a list of Saved Queries": {
+            "$ref": "./examples/SavedQueryList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroup/{resourceGroupName}/providers/Microsoft.ResourceGraph/queries/{resourceName}": {
+      "get": {
+        "description": "Get a single saved query by its resourceName.",
+        "operationId": "SavedQuery_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SavedQueryResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A saved query definition.",
+            "schema": {
+              "$ref": "#/definitions/SavedQueryResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/SavedQueryError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Saved Query": {
+            "$ref": "./examples/SavedQueryGet.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Delete a saved query.",
+        "operationId": "SavedQuery_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SavedQueryResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
>201 [](start = 11, length = 3)

As per [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#delete-resource](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#delete-resource), this should be 200.